egs/csj/s5/RESULTS

```diff
@@ -1,134 +1,212 @@
-%WER 12.86 [ 2211 / 17189, 338 ins, 378 del, 1495 sub ] exp/tri4_mmi_b0.1/decode_eval3_3.mdl_csj/wer_13_0.5
-%WER 12.98 [ 2231 / 17189, 331 ins, 380 del, 1520 sub ] exp/tri4_mmi_b0.1/decode_eval3_4.mdl_csj/wer_13_0.5
-%WER 9.97 [ 1713 / 17189, 239 ins, 281 del, 1193 sub ] exp/dnn5b_pretrain-dbn_dnn/decode_eval3_csj/wer_14_0.0
-%WER 9.65 [ 1658 / 17189, 237 ins, 271 del, 1150 sub ] exp/dnn5b_pretrain-dbn_dnn_smbr/decode_eval3_csj/wer_14_1.0
-%WER 9.29 [ 1597 / 17189, 236 ins, 249 del, 1112 sub ] exp/dnn5b_pretrain-dbn_dnn_smbr_i1lats/decode_eval3_csj/wer_14_1.0
+%WER 23.60 [ 4056 / 17189, 465 ins, 860 del, 2731 sub ] exp/tri1/decode_eval3_csj/wer_15_0.0
+%WER 22.66 [ 3895 / 17189, 506 ins, 782 del, 2607 sub ] exp/tri2/decode_eval3_csj/wer_14_0.0
+%WER 16.67 [ 2866 / 17189, 368 ins, 597 del, 1901 sub ] exp/tri3/decode_eval3_csj/wer_14_0.5
+%WER 17.61 [ 3027 / 17189, 421 ins, 561 del, 2045 sub ] exp/tri3/decode_eval3_csj_nosp/wer_14_0.0
+%WER 13.92 [ 2393 / 17189, 307 ins, 445 del, 1641 sub ] exp/tri4/decode_eval3_csj/wer_17_0.5
+%WER 17.91 [ 3079 / 17189, 434 ins, 540 del, 2105 sub ] exp/tri4/decode_eval3_csj.si/wer_15_0.5
+%WER 13.28 [ 2283 / 17189, 315 ins, 397 del, 1571 sub ] exp/tri4_fmmi_b0.1/decode_eval3_it4_csj/wer_14_0.5
+%WER 12.57 [ 2161 / 17189, 282 ins, 391 del, 1488 sub ] exp/tri4_fmmi_b0.1/decode_eval3_it5_csj/wer_14_0.5
+%WER 12.50 [ 2149 / 17189, 308 ins, 372 del, 1469 sub ] exp/tri4_fmmi_b0.1/decode_eval3_it6_csj/wer_13_0.5
+%WER 12.42 [ 2135 / 17189, 289 ins, 384 del, 1462 sub ] exp/tri4_fmmi_b0.1/decode_eval3_it7_csj/wer_11_1.0
+%WER 12.29 [ 2112 / 17189, 288 ins, 388 del, 1436 sub ] exp/tri4_fmmi_b0.1/decode_eval3_it8_csj/wer_12_1.0
+%WER 12.86 [ 2211 / 17189, 277 ins, 400 del, 1534 sub ] exp/tri4_mmi_b0.1/decode_eval3_1.mdl_csj/wer_14_0.5
+%WER 12.60 [ 2165 / 17189, 254 ins, 437 del, 1474 sub ] exp/tri4_mmi_b0.1/decode_eval3_2.mdl_csj/wer_13_1.0
+%WER 12.50 [ 2148 / 17189, 264 ins, 425 del, 1459 sub ] exp/tri4_mmi_b0.1/decode_eval3_3.mdl_csj/wer_13_1.0
+%WER 12.36 [ 2124 / 17189, 270 ins, 430 del, 1424 sub ] exp/tri4_mmi_b0.1/decode_eval3_4.mdl_csj/wer_13_1.0
+%WER 8.41 [ 1446 / 17189, 191 ins, 219 del, 1036 sub ] exp/chain/tdnn_csj/decode_eval3/wer_11_0.5
+%WER 8.38 [ 1440 / 17189, 177 ins, 224 del, 1039 sub ] exp/chain/tdnn_csj_online/decode_eval3/wer_10_1.0
 
-# nnet3 TDNN
+# nnet3 TDNN (former tdnn results)
 %WER 11.15 [ 2901 / 26028, 297 ins, 697 del, 1907 sub ] exp/nnet3/nnet_tdnn/decode_eval1_csj/wer_15_0.0
 %WER 8.47 [ 2259 / 26661, 289 ins, 381 del, 1589 sub ] exp/nnet3/nnet_tdnn/decode_eval2_csj/wer_12_0.0
-%WER 9.21 [ 1583 / 17189, 240 ins, 258 del, 1085 sub ] exp/nnet3/nnet_tdnn/decode_eval3_csj/wer_12_0.0
\ No newline at end of file
+%WER 9.21 [ 1583 / 17189, 240 ins, 258 del, 1085 sub ] exp/nnet3/nnet_tdnn/decode_eval3_csj/wer_12_0.0
+
+
+
+##### These experiments use the below line in csj_data_prep.sh
+## cat $CSJ/*/*/*-wav.list 2>/dev/null | sort > $dir/wav.flist # Using All data
```

Comment:
I think it would be better if instead of relying on someone editing csj_data_prep.sh, you added an option to csj_data_prep.sh and put one version of the invocation of that script (hopefully the better version) in the  run.sh, and the other version commented out in the run.sh.

Reply:
updated